File: webapp-java/dxa-module-core/src/main/java/com/sdl/dxa/modules/core/model/entity/Paragraph.java
package com.sdl.dxa.modules.core.model.entity;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.sdl.webapp.common.api.model.RichText;
import com.sdl.webapp.common.api.model.entity.AbstractEntityModel;
import com.sdl.webapp.common.api.model.entity.MediaItem;
import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper = true)
public class Paragraph extends AbstractEntityModel {

    @JsonProperty("Subheading")
    private String subheading;

    @JsonProperty("Content")
    private RichText content;

    @JsonProperty("Media")
    private MediaItem media;

    @JsonProperty("Caption")
    private String caption;
}

File: webapp-net/Core/Models/Entity/Paragraph.cs
using Sdl.Web.Common.Models;
using System;
namespace Sdl.Web.Modules.Core.Models
{
    [Serializable]
    public class Paragraph : EntityModel
    {
        public string Subheading { get; set; }
        public RichText Content { get; set; }
        public MediaItem Media { get; set; }
        public string Caption { get; set; }
    }
}